pnpm-workspace.yaml

```
@@ -44,6 +44,7 @@ patchedDependencies:
 onlyBuiltDependencies:
   - dprint
   - esbuild
+  - unrs-resolver
 
 ignoredBuiltDependencies:
   - core-js
```

It looks like this `postinstall` is only for legacy `npm` version. See: https://github.com/unrs/unrs-resolver/issues/56

We should instead ignore the `postinstall` script.

```suggestion

ignoredBuiltDependencies:
  - core-js
  - unrs-resolver
```